examples/example_pubsub_topics/main.py

```diff
@@ -29,7 +29,7 @@ def publish(request):
 
 
 # Triggered by pubsub topic. Simulates failure to trigger DLQ
-@app.pubsub_subscription("goblet-created-test-topic", dlq=True)
+@app.pubsub_subscription("goblet-created-test-topic", dlq=True, dlq_alert=True)
```

Comment:
lets add to the example `trigger_value` since the default value is zero, we should always include `trigger_value`

Reply:
Zero is enough for the DLQ messages because is going to trigger if the DLQ is not empty. I leaved that customizable maybe if we want to just alert if the DLQ is up to 10 or something like that. But will add to the example how to modify that